evm/src/test/java/org/hyperledger/besu/evm/code/EOFLayoutTest.java

```diff
@@ -295,6 +310,7 @@ void test(
       final String failureReason,
       final int expectedVersion) {
     final Bytes container = Bytes.fromHexString(containerString.replace(" ", ""));
+    System.out.println(container.toHexString());
```

Comment:
System.out.println ?

Reply:
I was dumping out the generated test cases for the fuzzing corpus.